pnpm-lock.yaml

```diff
@@ -4,18 +4,6 @@ settings:
   autoInstallPeers: true
   excludeLinksFromLockfile: false
 
-catalogs:
-  default:
-    next:
-      specifier: 14.2.14
-      version: 14.2.14
-    viem:
-      specifier: ^2.21.22
-      version: 2.21.22
-    wagmi:
-      specifier: ^2.12.17
-      version: 2.12.17
-
```

Comment:
why catalogs are removed?

Reply:
Please make sure we're generating the lockfile using the correct pnpm version specified in the pacakge.json

Reply:
Thx. Fixed after synced local pnpm version.